doc/pages/FAQ.md

```diff
@@ -87,8 +87,9 @@ You can manually disable package build sandboxing and remove bwrap from
 the required dependencies (at your own risk). You can use the built-in
 configuration as a template to create or update an `opamrc` file: run `opam
 init --show-default-opamrc >~/.opamrc`, then edit that file and remove or
-modify the `init-scripts:` and `wrap-*:` fields as well as the `bwrap` line
-from the `required-tools:` field, and finally retry `opam init`.
+modify the [`init-scripts:`](Manual.html#opamrcfield-init-scripts) and
+`wrap-*:` fields as well as the `bwrap` line from the `required-tools:` field,
+and finally retry `opam init`.
```

Comment:
Hmm, now there is `--disable-sandboxing`. But not sure if we want to advertise it, or even if it'll remain...

Reply:
imho, it shouldn't be in this FAQ field. Maybe another field at bottom, targeted for CI needs?

Reply:
Yeah... on the other hand, I feel uncomfortable with willingly providing confusing information when there is an easier way... so we'd better remove the explanation of `opamrc` altogether ?

Reply:
If we remove also this explanation, some people could be stuck: users that are not admin, or just can't install `bwrap` (exotic install).